misc/dist/html/full-size.html

```diff
@@ -88,7 +88,7 @@
 		</noscript>
 
 		<div id="status">
-			<img id="status-splash" src="$GODOT_SPLASH" alt="">
+			<img id="status-splash" src="$GODOT_SPLASH_IMAGE" alt="">
```

Comment:
```suggestion
			<img id="status-splash" src="$GODOT_SPLASH" alt="">
```

See my other comment.